smokey/src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java

```diff
@@ -224,20 +224,28 @@ public Rotation2d getGyroscopeRotation() {
     // 'getFusedHeading' or if it uses the 'getYaw' method (e.g., if isMagnetometerCalibrated() or not)
     // to run this test all we need is for someone to comment out the System.out.println lines of code below
 
+    /*
+    // This code does not work for us because 
```

Comment:
We might want to update this block to describe more fully why we are commenting it out for now.  This way next year when we revisit it we know oh yeah Navx2 will hit this block, Navx1 not so much.

Reply:
added more commentary.